boards/DIM/Test/Src/Test_StateMachine.cpp

```diff
@@ -179,7 +179,7 @@ class DimStateMachineTest : public testing::Test
     struct RotarySwitch *     drive_mode_switch;
 };
 
-TEST_F(DimStateMachineTest, check_drive_state_is_broadcasted_over_can)
+TEST_F(DimStateMachineTest, check_drive_state_is_broadcasted_over_can_DIM_12)
```

Comment:
I thought about leaving the ID as a comment, but it seems more concise to just leave the ID in the test case name

Reply:
Either one is fine by me.

Reply:
Actually after considering the ambiguity you pointed out (e.g. single vs double underscore, misinterpreting multiple IDs as a contiguous range of IDs), I think it's better to leave things as comments.